novatel_gps_driver/launch/tester_for_eth.launch

```diff
@@ -2,7 +2,7 @@
 <launch>
   <node name="novatel"
         pkg="nodelet" type="nodelet"
-        args="standalone novatel_gps_driver/novatel_gps_nodelet">
+        args="standalone novatel_gps_driver/novatel_gps_nodelet" output="screen" >
```

Comment:
Setting output="screen" will disable logging to a file. This is fine for development, but now that you are done you should remove this.

Reply:
yes, you are right.